src/randoop/main/Help.java

```diff
@@ -5,13 +5,13 @@
 import java.util.Arrays;
 import java.util.List;
 
+import randoop.Globals;
```

Comment:
You did not yet re-re-order the imports for this .java file, though.  (When you do so, the comment will be removed from the view in GitHub.)